rust/core-lib/src/editor.rs

```diff
@@ -169,7 +169,8 @@ impl<W: Write + Send + 'static> Editor<W> {
         self.views.len() > 0
     }
 
-    pub fn set_path<P: AsRef<Path>>(&mut self, path: P) {
+    /// should only ever be called from `BufferContainerRef::set_path`
+    pub fn _set_path<P: AsRef<Path>>(&mut self, path: P) {
```

Comment:
What is the purpose of the prepended underscore - to communicate that the API should not be used, but you still need it `pub` to be able to call it from `BufferContainerRef`?